dockers/actions-plan-preview/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM golang:1.16.5-alpine3.14 as builder
 COPY . /app
 RUN cd /app && go build -o /plan-preview .
 
-FROM gcr.io/pipecd/pipectl:v0.13.0
+FROM gcr.io/pipecd/pipectl:v0.13.1-4-gec4b78b
```

Comment:
I'm a bit worried here 👀  How about adding a TODO here or create an issue to remind us to update this Pipectl version of `v0.13.2` once it's rolled out?

Reply:
I'm on the train now so can you give me a suggestion to apply from my phone. 😀

Reply:
```suggestion
# TODO: Update pipectl version to v0.13.2 once it's rolled out.
FROM gcr.io/pipecd/pipectl:v0.13.1-4-gec4b78b
```
🙌

Reply:
Thank you. Applied it!

Reply:
always fantastic to see those helping each other out.